docs/usage/node-readiness.md

```diff
@@ -1,3 +1,7 @@
+---
+title: Marking Node-Critical Components and csi-driver-node Pods
```

Comment:
Here I chose `Marking Node-Critical Components and csi-driver-node Pods` because this title points to an action that the user might need to do as described in [the file section](https://github.com/gardener/gardener/blob/master/docs/usage/node-readiness.md#marking-node-critical-components). `Readiness of Shoot Worker Nodes` reads more as describing the concept of `Readiness` and does not suggest possible user actions while my proposal points to possible user action